text-inputs/src/main/res/values/styles-text-inputs.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="Widget.Vitamin.TextInputLayout" parent="Widget.Vitamin.TextInputLayout.FilledBox" />
```

Comment:
Maybe add regions for Filled and Outline to improve file readability?

Reply:
We defined `Widget.Vitamin.TextInputLayout.OutlinedBox` and `Widget.Vitamin.TextInputLayout.FilledBox` but this one is the default style applied for style attribute `textInputStyle`.